Week3Ses1/Week3Ses1/Program.cs

```diff
@@ -0,0 +1,205 @@
+﻿// See https://aka.ms/new-console-template for more information
+using System;
+using System.Linq.Expressions;
+
+Console.WriteLine("-------Ex1---------");
+int[] numbers = new int[]{1, 3, 2, 6, 12};
+int sum = 0;
+for (int i = 0; i < numbers.Length; i++)
+{
+    if (numbers[i] %2 ==0) { sum = sum+ numbers[i];}
+}
+Console.WriteLine("Suma nr pare este " + sum);
```

Comment:
Let's use only English, please. It will be easier for the future developer in you if you get used to English now, in the beginning. :)